apps/web/pages/pipelines.tsx

```diff
@@ -57,114 +39,164 @@ export default function ConnectionsScreen() {
-        </ul>
-        <div
-          className={twMerge(
-            'flex flex-1 flex-col gap-4',
-            pipelineType !== 'destinations' && 'hidden md:block',
-          )}>
-          <h2 className="hidden p-3 text-lg font-bold md:block">
-            Pipelines out
-          </h2>
-          {destinations.map((conn) => (
-            <ConnectionCard key={conn.id} connection={conn} />
-          ))}
+            <h2 className="p-3 text-lg font-medium">Pipelines out</h2>
+          </div>
+          <div className="mt-4 ml-11 flex flex-1 flex-col gap-4 pr-10">
+            {destinations.length == 0 ? (
+              <EmptyDestinationsView />
+            ) : (
+              destinations.map((conn) => (
+                <ConnectionCard key={conn.id} connection={conn} />
+              ))
+            )}
+          </div>
         </div>
       </div>
     </>
   )
 
   return (
-    <PageContainer
-      authenticated
-      flex
-      links={[
-        {
-          label: 'New connection',
-          href: '/pipelines?mode=connect',
-          primary: true,
-          fixed: true,
-        },
-      ]}>
-      {/*
-        We need this workaround so connect does not capture the wrong scope
-        TODO: Figure out a better way when ledgerId is still being loaded
-       */}
+    // TODO: Figure out a better way when ledgerId is still being loaded
+    <PageContainer title="Pipelines" flex>
+      {/* We need this workaround so connect does not capture the wrong scope */}
       {mode === 'connect' && connectWith.destinationId ? (
         <NewConnectionScreen connectWith={connectWith} />
       ) : (
-        myConnections
+        pipelinesInAndOut
       )}
     </PageContainer>
   )
 }
+
+function EmptySourcesView() {
+  return (
+    <div className="card max-w-md border border-base-content/25 bg-primaryUIControl p-10">
+      <p className="max-w-sm text-center text-sm">
+        Venice has over 10,000 financial data sources to choose from (e.g. your
+        bank)
+        <br />
+        <br />
+        Don&apos;t see one you need?&nbsp;
+        <Link
+          className="text-green"
+          href="mailto:hello@venice.is"
+          target="_blank">
+          Reach out!
+        </Link>
+      </p>
+    </div>
+  )
+}
+
+function EmptyDestinationsView() {
+  return (
+    <div className="card max-w-md border border-base-content/25 bg-primaryUIControl p-6">
+      <p className="max-w-sm text-center text-sm">
+        Your data is safe on Venice, but soon you&apos;ll be able to pipe it
+        into other destinations like Coda or Excel.
+        <br />
+        <br />
+        Have a destination in mind?&nbsp;
+        <Link
+          className="text-green"
+          href="mailto:hello@venice.is"
+          target="_blank">
+          Reach out!
+        </Link>
+      </p>
+    </div>
+  )
+}
+
+// Ledgers UI code (not in MVP for now)
+// TODO: Move this into settings screen (rarely used)
+
+// import {
```

Comment:
Note: ledger UI code is commented out for now; this should get moved into a debug menu somewhere until we're ready to enable it for users